packages/app/components/UsersTabContent.tsx

```diff
@@ -0,0 +1,188 @@
+                    });
+                    setUsers(owners);
+                })
+                .finally(() => setShowLoading(false));
+            });
+        }
+    }, [selectedOption, setUsers]);
+
+    useEffect(() => {
+        // Get details of all memberships to populate dropdown
+        if(mAddresses.length > 0) {
+            setShowLoading(true);
+            const promises = mAddresses.map((mAddr) => {
+                const contract = new web3.eth.Contract(getAbiFromJson(passportJson));
+                contract.options.address = mAddr;
+                return (contract.methods.get() as ContractSendMethod).call().then((p) => {
+                    const data: MembershipDetailMap = {};
+                    data[mAddr] = {
+                        name: p[0],
+                        symbol: p[1],
+                        supply: p[2],
+                        price: web3.utils.fromWei(p[3], "ether"),
+                    };
+                    return data;
+                });
+            });
+            Promise.all(promises).then((values) => {
+                const data = Object.assign({}, ...values);
+                setMembershipDetails(data);
+            })
+            .finally(() => setShowLoading(false));;
+        }
+        else {
+            setMembershipDetails({});
+        }
+    }, [mAddresses]);
+
+    return (
+        <>
+            <h1>Users</h1>
+            <SmallBox>
+                {(Object.keys(membershipDetails).length > 0) ?
+                    <Select
+                        label = "Choose Membership Type:"
+                        options = {Object.keys(membershipDetails).map((addr) => {
+                            return {
+                                key: addr, 
+                                label: `${membershipDetails[addr]["name"]} (${membershipDetails[addr]["symbol"]})`
+                            }
+                        })}
+                        onChange = {(val) => setSelectedOption(val)}
+                        disabled = {false}
+                    />  : <div>Please create some Membership Types first!</div>
+                }
+            </SmallBox>
+            { showLoading ? 
+                (<Label 
+                    label={`Loading...`}
+                />) : null
+            }
+            <TableBox>
+            {selectedOption ?
+                (<>
+                    <Label 
+                        label={`Remaining supply: ${membershipDetails[selectedOption]['supply']}`}
+                    />
+                    {Object.keys(users).length>0 ?
+                        (<Table>
+                            <tr>
+                                <TableHeader>Address</TableHeader>
+                                <TableHeader>Passports owned</TableHeader>
+                            </tr>
+                            {
+                                Object.keys(users).map((user) => {
+                                    return (<tr>
```

Comment:
Deployment failed here, will need to address this first

![image](https://user-images.githubusercontent.com/7143571/151667876-5d5a1e47-e749-4e71-be04-2e669b50e810.png)

Reply:
My bad....forgot to run the linter. Fixed now.